deepcell/data/tracking_test.py

```diff
@@ -129,52 +125,51 @@ def test_random_translate(self, time, max_cells, crop_size,
         self.assertAllEqual(output_X['centroids'], X_cents)
 
     def test_prepare_dataset(self, time, max_cells, crop_size,
-                             batch_size, seed, track_length, val_split):
+                             batch_size, seed, track_length,
+                             val_split, test_split):
         # Create track_info and prepare the dataset
         track_info = self.create_track_info(226, time, max_cells, crop_size)
-        train_data, val_data = tracking.prepare_dataset(track_info, rotation_range=180,
-                                                        batch_size=batch_size, seed=seed,
-                                                        track_length=track_length,
-                                                        val_split=val_split)
+        train_data, val_data, test_data = tracking.prepare_dataset(track_info,
+                                                                   rotation_range=180,
+                                                                   batch_size=batch_size,
+                                                                   seed=seed,
+                                                                   track_length=track_length,
+                                                                   val_split=val_split,
+                                                                   test_split=test_split)
         # Test data correctly batched
         for X, y in train_data.take(1):
             self.assertEqual((X['appearances'].shape)[0], batch_size)
         for X, y in val_data.take(1):
             self.assertEqual((X['appearances'].shape)[0], batch_size)
+        if not test_split == 0:
```

Comment:
Slightly clearer

```suggestion
        if test_split != 0:
```

Reply:
What happens if it is 0? Will there be an exception raised? Let's add a test for the `else` clause here.

Reply:
@R-Ding This is especially important as we specifically set `test_split = 0` as default

Reply:
I am having trouble finding a way to test whether `test_data` is empty when `test_split=0`, especially because after applying `.filter()` to any dataset, the cardinality becomes -2. Do either of you have any suggestions?